patches/0005-Prefer-Quilt-over-Fabric-Loader.patch

```diff
@@ -102,8 +82,7 @@ index 0000000000000000000000000000000000000000..a0b77bedca33b58cc7112002dae4527c
 +		var id = componentMetadataContext.getDetails().getId();
 +		if (id.getGroup().equals("org.quiltmc") && id.getName().equals("quilt-loader")) {
 +			componentMetadataContext.getDetails().allVariants(
-+					// TODO: actually provide the version here?
```

Comment:
This TODO probably shouldn't have been removed, since this isn't fixed. Once loader adds a capability to its component metadata, a possible approach might be a hardcoded version range list in loom or somewhere on the meta or maven server for versions older than that.
This would currently fail when loader adds the capability, too, see https://docs.gradle.org/8.10/javadoc/org/gradle/api/capabilities/MutableCapabilitiesMetadata.html#addCapability(java.lang.String,java.lang.String,java.lang.String)

Reply:
Honestly? I do wonder if you can add the capability conditionally; but yeah, don't worry about it; we do have plans to tackle it Loader-side; I do worry about the possibility of hardcoding such a version range list, but oh well

Reply:
Well, you have to add it conditionally, otherwise doing so loader-side becomes impossible. getCapabilities would be the way to go.